### ISSUE TITLE
[Feature request] Allow specifying `changepoint_ds_lower` and `changepoint_ds_upper`

### ISSUE THREAD

**Issue by tcuongd** [facebook/prophet#1987]
Do we think it would be useful to allow end users to specify the changepoint range in terms of the dates of the data, rather than the percentage of history (the existing `changepoint_range`)?

I think the default value of `changepoint_range` can have a huge impact on the forecast (e.g. when the time series shows recent growth), but goes largely unnoticed (e.g. as we collect more historical data, the first 80% of history becomes less relevant). Setting a percentage seems a bit more confusing compared to setting a specific dates.

We would add these arguments directly to the `__init__` method, and make the required changes to `self.set_changepoints()`. I understand that the `__init__` methods are already super bloated as is though, so keen to know if people think the added convenience of this feature is worth it.

**Comment by joanna-baran-allegro** [comment]
Yes, that's exactly what I needed for the project. It would be good to fix this param in such a way that the last specified period of train data will be without trend changepoints (e.g. last 4 weeks)
